sale_order_delivery_products/views/sale_order_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Daniel Dominguez, Manuel Calero - https://xtendoo.es
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.delivery_product</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_quotation_send']" position="after">
+                <button
+                    string="Deliver Products"
```

Comment:
Is that really 'delivery' pickings ? As in several steps configuration, you can have Pick and Pack ones too